### ISSUE TITLE
Enderchest Shortcut

### ISSUE THREAD

**Issue by pixo2000** [Jannyboy11/InvSee-plus-plus#132]
Just that the slot in the bottom right in Inventories is an Enderchest as shortcut to the players ec

**Comment by Jannyboy11** [comment]
It would be inconsistent with the enderchest spectator inventory - there is no room for a slot to return to the main inventory.

**Comment by pixo2000** [comment]
yeah. i dont need one but i would love just having a shortcut for it. idk if it is possible, but if i press escape to close it again, maybe reopen the inventory?

**Comment by Jannyboy11** [comment]
I'm going to shelve this idea. I think such functionality is better implemented in some kind of general player management plugin.

**Comment by pixo2000** [comment]
Thanks! Do you know any specific management plugin?

Jan ***@***.***> schrieb am Mo., 16. Dez. 2024, 22:12:

**Comment by Jannyboy11** [comment]
The only one I know of is CMI, but it does a lot more than just player management.
